webrender/src/device.rs

```diff
@@ -592,32 +594,50 @@ impl ProgramSources {
     }
 }
 
+#[cfg_attr(feature = "serialize_program", derive(Deserialize, Serialize))]
 pub struct ProgramBinary {
+    sources: ProgramSources,
```

Comment:
can we only have this field for when `serialize_program` is featured?

Reply:
Yes, I am going to address it.